app/main/views/frameworks.py

```diff
@@ -725,39 +724,30 @@ def framework_updates_email_clarification_question(framework_slug):
     # Submit email to Zendesk so the question can be answered
     # Fail if this email does not send
     if framework['clarificationQuestionsOpen']:
-        subject = "{} clarification question".format(framework['name'])
         to_address = current_app.config['DM_CLARIFICATION_QUESTION_EMAIL']
-        from_address = current_app.config['DM_ENQUIRIES_EMAIL_ADDRESS']
-        email_body = render_template(
-            "emails/clarification_question.html",
-            supplier_id=current_user.supplier_id,
-            message=clarification_question
-        )
-        tags = ["clarification-question"]
+        personalisation = {
+            "framework_name": framework['name'],
+            "supplier_id": current_user.supplier_id,
+            "clarification_question": clarification_question
+        }
+        template = 'framework-clarification-question'
     else:
-        subject = "{} application question".format(framework['name'])
         to_address = current_app.config['DM_FOLLOW_UP_EMAIL_TO']
-        from_address = current_user.email_address
-        email_body = render_template(
-            "emails/follow_up_question.html",
-            supplier_name=current_user.supplier_name,
-            user_name=current_user.name,
-            user_email=current_user.email_address,
-            framework_name=framework['name'],
-            message=clarification_question
-        )
-        tags = ["application-question"]
+        personalisation = {
+            "framework_name": framework['name'],
+            "supplier_name": current_user.supplier_name,
+            "user_name": current_user.name,
+            "user_email": current_user.email_address,
+            "application_question": clarification_question
+        }
+        template = "framework-application-question"
 
-    mandrill_client = DMMandrillClient()
+    notify_client = DMNotifyClient()
     try:
-        mandrill_client.send_email(
-            to_email_addresses=to_address,
-            email_body=email_body,
-            subject=subject,
-            from_email_address=current_app.config["DM_ENQUIRIES_EMAIL_ADDRESS"],
-            from_name="{} Supplier".format(framework['name']),
-            tags=tags,
-            reply_to=from_address,
+        notify_client.send_email(
+            to_email_address=to_address,
+            template_name_or_id=notify_client.templates[template],
```

Comment:
I think looking up the template name here is strictly unnecessary, it could equally be

```suggestion
            template_name_or_id=template,
```

Reply:
Ah yes, you're right!